CSharp_Course_Project/Program.cs

```diff
@@ -4,9 +4,442 @@ namespace CSharp_Course_Project
 {
     class Program
     {
-        static void Main(string[] args)
+        public static void SecondsAndHours()
+        {
+            int seconds = Convert.ToInt32(Console.ReadLine());
+            int hours = seconds / 3600;
+            Console.WriteLine(hours);
+        }
+
+        public static void ThreeNumbers()
         {
+            int a = Convert.ToInt32(Console.ReadLine());
+            int b = Convert.ToInt32(Console.ReadLine());
+            int c = Convert.ToInt32(Console.ReadLine());
+
+            if ((b > a && b < c) || (b > c && b < a))
+            {
+                Console.WriteLine("B is between A and C");
+            }
+            else
+            {
+                Console.WriteLine("B is not betweeen A and C");
+            }
+        }
+
+        public static void OddAndThreeDigitNumber()
+        {
+            string n = Console.ReadLine();
+            int number = Convert.ToInt32(n);
+
+            if (n.Length == 3 && number % 2 != 0)
```

Comment:
It is cheating :).
I thought you define the digit order and count the length :).